python/lsst/ts/m2gui/main_window.py

```diff
@@ -434,7 +434,7 @@ def _create_dialog_exit(self) -> QMessageBoxAsync:
 
         Returns
         -------
-        `QMessageBoxAsync`
+        dialog : `QMessageBoxAsync`
```

Comment:
You removed the return names of a few functions on `tab_diagnostics.py` (I assume, based on [this](https://numpydoc.readthedocs.io/en/latest/format.html#returns)) so you might want to keep the same standard here?

Reply:
This function here does do the return:

```
return dialog
```

The removed parts in `tab_diagnostics.py` do not.

Thanks!